patches/sdk/0009-PlatformAbstractions-is-not-produced-in-source-build.patch

```diff
@@ -0,0 +1,31 @@
+From 85a0f4e16ec8840eb6a76a043c48c079e4c73348 Mon Sep 17 00:00:00 2001
+From: Chris Rummel <crummel@microsoft.com>
+Date: Wed, 12 Aug 2020 14:26:52 -0500
+Subject: [PATCH 9/9] PlatformAbstractions is not produced in source-build
+
+---
+ .../Microsoft.NET.Build.Tasks.csproj                          | 4 ++--
+ 1 file changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/src/Tasks/Microsoft.NET.Build.Tasks/Microsoft.NET.Build.Tasks.csproj b/src/Tasks/Microsoft.NET.Build.Tasks/Microsoft.NET.Build.Tasks.csproj
+index 94467319e..7c565a36e 100644
+--- a/src/Tasks/Microsoft.NET.Build.Tasks/Microsoft.NET.Build.Tasks.csproj
++++ b/src/Tasks/Microsoft.NET.Build.Tasks/Microsoft.NET.Build.Tasks.csproj
+@@ -1,4 +1,4 @@
+-﻿<?xml version="1.0" encoding="utf-8"?>
++﻿<?xml version="1.0" encoding="utf-8"?>
+ <!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the MIT license. See License.txt in the project root for full license information. -->
+ <Project>
+ 
+@@ -47,7 +47,7 @@
+     <PackageReference Include="Microsoft.Build.Framework" Version="15.7.179" ExcludeAssets="Runtime" />
+     <PackageReference Include="Microsoft.Build.Utilities.Core" Version="15.7.179" ExcludeAssets="Runtime" />
+     <PackageReference Include="Microsoft.Extensions.DependencyModel" Version="$(MicrosoftExtensionsDependencyModelVersion)" />
+-    <PackageReference Include="Microsoft.DotNet.PlatformAbstractions" Version="$(MicrosoftDotNetPlatformAbstractionsPackageVersion)" />
++    <PackageReference Include="Microsoft.DotNet.PlatformAbstractions" Version="2.1.0" />
```

Comment:
If we aren't producing it, why doesn't the default value work, why is it 2.1.0 here?

Reply:
Should we fix it in the original patch then where we add it? (I was wondering how this would work in the Microsoft build if there is no default and saw that we're adding it in the broken state.)

https://github.com/dotnet/source-build/blob/d697561c4b4bcd312aab23da573eaf9e7453d467/patches/sdk/0001-Add-missing-PlatformAbstractions-reference.patch#L18